tests/codegen/ccode/scripts/arrays_create.py

```diff
@@ -1,5 +1,4 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring/
-from pyccel.decorators import types
+# pylint: disable=missing-function-docstring, missing-module-docstring, unused-variable/
```

Comment:
Please don't disable errors for an entire file. The only exception to this rule is disabling missing-docstring in test files

Reply:
fixed